pages/blog/integrating-mailchimp-with-your-nodejs-app.mdx

```diff
@@ -0,0 +1,350 @@
+
+1. MailChimp Account API Key
+2. Audience ID
+
+### Getting your API Key
+
+For integrating your app, the first and most important thing needed is an API Key. This is a relatively easy step; have a look at the steps below.
+
+1. Go to the bottom left corner of the screen and click on the **Account** option.
+
+<Image
+  src="integrating-mailchimp-with-your-nodejs-app/mc1.png"
+  alt="MailChimp Profile"
+  width={216}
+  height={94}
+/>
+<Image
+  src="integrating-mailchimp-with-your-nodejs-app/mc2.png"
+  alt="MailChimp Account Menu"
+  width={297}
+  height={320}
+/>
+
+2. On the Account page, you’ll find a menu. Click on **Extras** and find the **API Keys** option.
+
+<Image
+  src="integrating-mailchimp-with-your-nodejs-app/mc3.png"
+  alt="API Key option in MailChimp Account"
+  width={157}
+  height={193}
+/>
+
+3. If you’re opening this up for the first time, you’ll see a screen similar to this one here. You need to have an API key to integrate your MailChimp account to your NodeJS application. To do this, click on **Create A Key**.
+
+<Image
+  src="integrating-mailchimp-with-your-nodejs-app/mc4.png"
+  alt="Generating API Keys"
+  width={1276}
+  height={603}
+/>
+
+4. Once you click on it, you’ll see that MailChimp has created an API key for you! It will look something similar to this here
+   <Image
+     src="integrating-mailchimp-with-your-nodejs-app/mc5.png"
+     alt="Active API Keys"
+     width={1268}
+     height={317}
+   />
+
+We have displayed the latter portion of our API Key in the image above. One thing to note here are the last three characters after the `-`. This represents the data center corresponding to your account, which is for all the API calls one can make. For us, it is us6. This might be different for you, so make sure to note this separately.
+
+### Finding your Audience ID
+
+Next up, we need to find our Audience ID. For this, just follow this 2 step process :)
+
+1. Go to the audience tab and click on **All contacts**.
+
+<Image
+  src="integrating-mailchimp-with-your-nodejs-app/mc6.png"
+  alt="MailChimp Audience Tab"
+  width={553}
+  height={499}
+/>
+
+2. You'll see a screen with all your contact details mentioned. Go to the settings tab and click on **Audience name and defaults**.
+
+<Image
+  src="integrating-mailchimp-with-your-nodejs-app/mc7.png"
+  alt="MailChimp Audience Settings"
+  width={338}
+  height={430}
+/>
+
+3. You can find your **Audience ID**, which will look something similar to this here!
+
+<Image
+  src="integrating-mailchimp-with-your-nodejs-app/mc8.png"
+  alt="MailChimp Audience ID"
+  width={483}
+  height={116}
+/>
```

Comment:
I believe this is a total beginners' guide so the handholding is much needed. I want the users to directly see this blog rather than anything else in the Mailchimp documentation If they're looking to integrate it with the Node.js app.

The aim of this blog was to capture SEO as I remember. If we're redirecting folks to Mailchimp documentation it will be worse for us generally I believe. There are already existing blogs that deep dive into the application perspective and even Mailchimp has excellent documentation of building this rather.

Also, @andyjih has this part in his sample blog.